django/db/models/fields/json.py

```diff
@@ -379,6 +380,18 @@ def as_mysql(self, compiler, connection):
             json_path = compile_json_path(key_transforms)
             return "(%s ->> %%s)" % lhs, tuple(params) + (json_path,)
 
+    @classmethod
+    def from_lookup(cls, lookup):
+        transform, *keys = lookup.split(LOOKUP_SEP)
+        if not keys:
+            raise ValueError("Missing keys")
```

Comment:
```suggestion
            raise ValueError("lookup cannot be empty.")
```

Reply:
Provide a non-empty `lookup` - giving only `transform` would produce this error. Wouldn't this be incorrect?

Reply:
Right, maybe _"lookup must contains key or index transforms."_ :thinking: .